bundles/org.openhab.binding.zm/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,86 @@
+		</parameter-group>
+		<parameter-group name="auth-info">
+			<label>Authentication Information</label>
+		</parameter-group>
+		<parameter name="refreshInterval" type="integer" min="2" required="false" unit="s" groupName="config-info">
+			<label>Refresh Interval (Normal)</label>
+			<description>Interval in seconds at which monitor status is updated</description>
+			<default>5</default>
+		</parameter>
+		<parameter name="discoveryEnabled" type="boolean" required="false" groupName="config-info">
+			<label>Discovery Enabled</label>
+			<description>Enable/disable automatic discovery</description>
+			<default>true</default>
+		</parameter>
+		<parameter name="discoveryInterval" type="integer" min="60" required="false" unit="s"
+			groupName="config-info">
+			<label>Monitor Discovery Interval</label>
+			<description>Specifies time in seconds in which the binding will attempt to discover monitors</description>
+			<default>300</default>
+		</parameter>
+		<parameter name="defaultAlarmDuration" type="integer" unit="s" required="false" groupName="config-info">
+			<label>Default Alarm Duration</label>
+			<description>Duration in seconds after which the alarm will be turned off</description>
+			<default>60</default>
+		</parameter>
+		<parameter name="defaultImageRefreshInterval" type="integer" required="false" groupName="config-info">
+			<label>Default Image Refresh Interval</label>
+			<description>Interval in seconds at which monitor image snapshot will be updated</description>
+		</parameter>
+		<parameter name="host" type="text" required="true" groupName="url-info">
+			<label>Server</label>
+			<description>Zoneminder server name or IP address</description>
+			<context>network-address</context>
+		</parameter>
+		<parameter name="portNumber" type="text" required="false" groupName="url-info">
+			<label>Port Number</label>
+			<description>Port Number</description>
+		</parameter>
+		<parameter name="useDefaultUrlPath" type="boolean" required="false" groupName="url-info">
+			<label>Use Default URL Path</label>
+			<description>Use standard URL path (i.e. /zm)</description>
+			<default>true</default>
+		</parameter>
+		<parameter name="urlPath" type="text" required="false" groupName="url-info">
+			<label>URL Path</label>
+			<description>URL path (Used if Zoneminder URL starts with something other than /zm)</description>
+		</parameter>
+		<parameter name="user" type="text" required="false" groupName="auth-info">
+			<label>User Name</label>
+			<description>User name (if authentication enabled in Zoneminder)</description>
+		</parameter>
+		<parameter name="pass" type="text" required="false" groupName="auth-info">
+			<label>Password</label>
+			<description>Password (if authentication enabled in Zoneminder)</description>
+			<context>password</context>
+		</parameter>
+	</config-description>
+
+	<config-description uri="thing-type:zm:monitor">
+		<parameter name="monitorId" type="text" required="true">
```

Comment:
Is this really text or integer?

Reply:
Zoneminder treats it as text, so I wanted to be consistent. Even though they appear to be all numeric now, I don't think it can be guaranteed to stay that way.